Prox/Prox/Models/PlaceProviders.swift

```diff
@@ -62,7 +62,7 @@ class SinglePlaceProvider: PlaceProvider {
         let photoURLStrings = (dict["images"] as? [[String:String]])?.flatMap { $0["src"] } ?? []
         photoURLs = photoURLStrings.flatMap { URL(string: $0) }
 
-        if let urlString = dict["url"] as? String,
+        if let urlString = dict["url"] as? String ?? dict["website"] as? String,
```

Comment:
IIRC, `website` is going to the be the "real" website (if it exists), whereas `url` will be the provider-specific website. If so, invert the order here?

It might also be worth having a separate `website` field in `PlaceProvider`, but I suppose we can add that when we need it.